dashboard/app/models/unit_group.rb

```diff
@@ -76,12 +76,10 @@ def localized_version_title
     I18n.t("data.course.name.#{name}.version_title", default: version_year)
   end
 
-  # Any course with a plc_course or no family_name is considered stable.
-  # All other courses must specify an is_stable boolean property.
+  # Any course with a plc_course is considered stable.
+  # All other courses must specify a published_state.
   def stable?
-    return true if plc_course || !family_name
-
-    is_stable || false
+    plc_course || published_state == SharedConstants::PUBLISHED_STATE.stable
```

Comment:
nit: can you put parenthesis around `published_state == SharedConstants::PUBLISHED_STATE.stable`? From first glance, this line is a bit ambiguous